README.md

```diff
@@ -164,6 +164,17 @@ connection_retries: 3
 log_level: 'warn'
 ```
 
+## Change Log
+All notable changes to QingStor SDK for Ruby will be documented here.
+
+### [v2.0.1] - 2016-12-16
+#### Changed
+- Improve the implementation of deleting multiple objects.
+
+### [v2.0.0] - 2016-12-14
```

Comment:
The first version don't need to add compare link
ref: https://github.com/olivierlacan/keep-a-changelog/blame/master/CHANGELOG.md#L92